addons/account/models/account_invoice_import_wizard.py

```diff
@@ -8,9 +8,11 @@ class ImportInvoiceImportWizard(models.TransientModel):
     _description = 'Import Your Vendor Bills from Files.'
 
     attachment_ids = fields.Many2many('ir.attachment', string='Files')
+    journal_id = fields.Many2one(string="Journal", comodel_name="account.journal", required=True, domain="[('type', '=', 'purchase')]", help="Journal where to generate the bills")
```

Comment:
default value should be filled
* when clicking 'import' from the dashboard: the one from the context (OK)
* when clicking on a journal in the dashboard to open the list view, then clicking on 'upload': the one from the context (NOK)
* when going in the vendor bills menuitem (all purchase journal displayed): the one qith the lowest sequence (NOK)